kolibri/content/serializers.py

```diff
@@ -131,7 +132,7 @@ class Meta:
         model = ContentNode
         fields = (
             'pk', 'content_id', 'title', 'description', 'kind', 'available', 'tags', 'sort_order', 'license_owner',
-            'license', 'files', 'ancestors', 'parent', 'thumbnail', 'progress_fraction', 'next_content', 'author',
+            'license', 'license_description', 'files', 'ancestors', 'parent', 'thumbnail', 'progress_fraction', 'next_content', 'author',
```

Comment:
There's no serializer specified for the `license_description` field here - would need to specify that we are getting this from the `license.description` lookup.